js/app.js

```diff
@@ -0,0 +1,27 @@
+const select = document.querySelector('select');
+const allLang = ['en', 'ua', 'ru'];
+
+select.addEventListener('change', changeURLLanguage);
+function changeURLLanguage(){
+	let lang = select.value;
+	location.href = window.location.pathname + '#'+lang
+	location.reload();
+}
+
+function changeLanguage(){

```

Comment:
Above I'm seeing an event listener for the URL change. But I'm not seeing the event listener for a change language function. Should it be here?